docs/dev/new_check_howto.md

```diff
@@ -315,7 +391,7 @@ The `logos` folder must contain **three** images with filenames and sizes that m
 * Event stream at `/event/stream`
 * Notification icons at `/report/monitor`
 
-### Metadata
+### Integration Metadata
```

Comment:
nit, feel free to ignore - isn't the context clear enough to avoid repeating "Integration" at every step?

Reply:
if we move this to another file, then we will remove the "integration" in each title.